tasks/main.yml

```diff
@@ -54,5 +54,5 @@
     state: absent
   loop:
     - /etc/apt/sources.list.d/netbird-wiretrustee.list
-    - /usr/local/src/netbird-wiretrustee-public.key
```

Comment:
As mentioned in https://github.com/Zorlin/ansible-role-netbird/pull/1, the line I removed here should stay for cleaning up outdated files 🤔